From ef64493b8913e4069c4422ad14da6de405c445f6 Mon Sep 17 00:00:00 2001
From: Jon Dufresne <jon.dufresne@gmail.com>
Date: Sun, 3 Dec 2017 18:47:19 -0800
Subject: [PATCH] Fix use of "async" in test_cursor.py

"async" will be a keyword starting with Python 3.7. On Python 3.6, use
of "async" causes a deprecation warning. Use the alias "async_" instead.
---
 tests/test_cursor.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_cursor.py b/tests/test_cursor.py
index e896ef94..5c94f422 100755
--- a/tests/test_cursor.py
+++ b/tests/test_cursor.py
@@ -553,7 +553,7 @@ def test_external_close_async(self):
         # Issue #443 is in the async code too. Since the fix is duplicated,
         # so is the test.
         control_conn = self.conn
-        connect_func = lambda: self.connect(async=True)
+        connect_func = lambda: self.connect(async_=True)
         wait_func = psycopg2.extras.wait_select
         self._test_external_close(control_conn, connect_func, wait_func)
 
